src/FileUtils/AgGridUtils.js

```diff
@@ -185,7 +186,7 @@ const fromCSV = (dataStr, hasHeader = true, cols, options) => {
     return { error: [err] };
   }
 
-  if (!cols) {
+  if (!cols || cols.length === 0) {
```

Comment:
`cols` is the result of `getFlattenColumnsWithoutGroups`, we know that it will be an Array (and thus not a bool or number) so we can allow a shorter syntax here :slightly_smiling_face: